resources/js/components/global/DashboardNotification.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="dashboard-notification padding-notification">
+    <div class="row">
+      <div class="dashboard-notification__message-container col-7">
```

Comment:
You don't need the class of ".dashboard-notification__message-container" here. Remove it and the CSS associated with it in your stylesheet.